code/__DEFINES/auxtools.dm

```diff
@@ -0,0 +1,9 @@
+#define AUXLUA (world.system_type == MS_WINDOWS ? "auxlua.dll" : __detect_auxtools("auxlua"))
+
+/proc/__detect_auxtools(library)
```

Comment:
I think you need to protect this or someone will get mad. Like checking if it is an advanced proc call or not.

Also this is not a define and should not be in defines.

Reply:
I copied the placement of the file from its equivalent on Bee, should it go in _HELPERS instead?

Reply:
Yes

Also autodoc the AUXLUA define since other files use it.